website_sale_osc/models/website.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+# Odoo, an open source suite of business apps
+# This module copyright (C) 2015 bloopark systems (<http://bloopark.de>).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp.models import Model
+from openerp import fields
+
+
+class website(Model):
+
+    """Adds the fields for options of the OSC."""
+
+    _inherit = 'website'
+
+    use_osc = fields.Boolean(
+        string='Use OSC',
+        default=True,)
+    use_all_checkout_countries = fields.Boolean(
```

Comment:
You don't need this field: if checkout_country_ids is empty, you assume all countries.